kernel_config

```diff
@@ -4822,3 +4822,7 @@ CONFIG_FONT_8x16=y
 CONFIG_ARCH_HAS_SG_CHAIN=y
 CONFIG_ARCH_HAS_PMEM_API=y
 CONFIG_ARCH_HAS_MMIO_FLUSH=y
+
+CONFIG_EXPERIMENTAL=y
+CONFIG_IPV6_ADVANCED_ROUTER=y
+CONFIG_ARPD=y
```

Comment:
Do these even exist for our version of the kernel?  I don't see any of them in Debian's 4.9 config (but we use 4.4).